Lib/test/test_array.py

```diff
@@ -1680,6 +1681,45 @@ def test_gh_128961(self):
         it.__setstate__(0)
         self.assertRaises(StopIteration, next, it)
 
+    # Tests for NULL pointer dereference in array.__setitem__
+    # when the index conversion mutates the array.
+    # See: github.com/python/cpython/issues/142555.
```

Comment:
```suggestion
    # See: https://github.com/python/cpython/issues/142555.
```

I also need the https:// part (sorry xD)